src/main/java/com/iotiq/application/service/ProductService.java

```diff
@@ -160,49 +160,81 @@ public byte[] exportCSVFile(UUID sellerId) throws IOException {
+                    product.getMaterialBehavior() != null ? product.getMaterialBehavior().getColorfast() : "",
+                    product.getMaterialBehavior() != null ? product.getMaterialBehavior().getMoistureTransporting() : "",
+                    product.getMaterialBehavior() != null ? product.getMaterialBehavior().getOdorNeutralizing() : "",
+                    product.getMaterialBehavior() != null ? product.getMaterialBehavior().getScratchResistant() : "",
+                    product.getMaterialBehavior() != null ? product.getMaterialBehavior().getSweatWicking() : "",
+                    product.getMaterialBehavior() != null ? product.getMaterialBehavior().getWashable() : "",
+                    product.getMaterialParameter() != null ? product.getMaterialParameter().getThickness() : "",
+                    product.getMaterialParameter() != null ? product.getMaterialParameter().getFlexibility() : "",
+                    product.getMaterialParameter() != null ? product.getMaterialParameter().getBreathability() : "",
+                    product.getMaterialParameter() != null ? product.getMaterialParameter().getMoistureWicking() : "",
+                    product.getMotif(),
+                    product.getNeurodermatitis(),
+                    product.getOekotexStandard(),
+                    product.getPriceRange(),
+                    product.getRating(),
+                    separateWithSemicolon(product.getSizes()),
+                    separateWithSemicolon((product.getSpecificBodyParts())),
+                    separateWithSemicolon(product.getSpecificFunctionalities()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getEnvironmentalCompatibilities() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getLifeCycles() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getRegionalityList() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getResourceConsumptions() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getSocialEthics() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getSustainabilityCompositions() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getSustainabilityLightweights() : List.of()),
+                    separateWithSemicolon(product.getSustainability() != null ? product.getSustainability().getSkills() : List.of()));
+        } catch (IOException e) {
+            throw new RuntimeException("Fehler beim Schreiben des CSV-Datensatzes.", e);
```

Comment:
Use a specific exception for this. Also get the exception error from translartions